client/src/partials/reports_proposed/variation_exploitation/variation_exploitation.html

```diff
@@ -0,0 +1,46 @@
+<header data-header>
+  {{'VARIATION_EXPLOITATION.BUILD_VARIATION_EXPLOITATION' | translate}}
+</header>
+
+<main>
+  <div class="panel panel-default">
+    <div class="panel-heading">{{'VARIATION_EXPLOITATION.DOCUMENT_CONFIG' | translate}}</div>
+      <div class="panel-body">
+        <form>
+
+          <div class="form-group">
+            <label class="control-label col-xs-4 required">
```

Comment:
You don't need the classes `control-label` or `col-xs-4`.  Those are only useful when you are creating a Horizontal or Inline form.

Also, the `col-xs-4` class makes the label not line up with the `<select>` or `<input>` element.  Here are two screenshots from reports, one that is a copy and paste of this code and another that was written without the `control-label` and `col-xs-4`.

#### Has col-xs tag
![col-xs-report](https://cloud.githubusercontent.com/assets/896472/10666724/55acc6fc-78cb-11e5-940c-4fc73e39de22.png)

#### No col-xs tag
![no-col-xs-report](https://cloud.githubusercontent.com/assets/896472/10666735/5fd451fe-78cb-11e5-9b58-d9297e455af5.png)

While it won't break the view, you can tell that the one missing the col-xs tag lines up much better with the input.